components/automatic_points_table/commons/automatic_points_table.lua

```diff
@@ -0,0 +1,106 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:AutomaticPointsTable
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+---- This module creates a Points Table that automatically gets data from the prizepools
+---- of set tournaments
+
+local Class = require('Module:Class')
+-- local Table = require('Module:Table')
+
+local DivTable = require('Module:DivTable')
+local Json = require('Module:Json')
+local split = require('Module:StringUtils').split
+local getArgs = require('Module:Arguments').getArgs
+
+function DivTable.Row:create()
+  for _, cell in pairs(self.cells) do
+      cell:css('border-right', '1px solid #bbbbbb')

```

Comment:
hard coded color
please add a class to css instead and use it here
(for darkmode support all hardcoded colors should be removed (replaced by css classes))

Reply:
There are so many hard-coded colors in the rest of the code I've realized and removing them breaks the style.
Where can I find the preexisting CSS classes?

Reply:
for bg/pbg `'bg-' .. string.lower(args['pbg' .. key])` should be the classes you want (i think they should all be set with important, else we will have to adjust it)

Reply:
for the css for the divtable i think
https://liquipedia.net/commons/MediaWiki:Common.css/DivTable.css
would be best

for the deductions i think
https://liquipedia.net/commons/MediaWiki:Common.css/Miscellaneous.css

Reply:
for colors we also have
https://liquipedia.net/commons/MediaWiki:Common.css/Colours.css
but those are mostly not set as important and the priority of that sheet isn't too high, so it mostly gets overwritten by other styles

Reply:
rest should probably go mostly into
https://liquipedia.net/commons/MediaWiki:Common.css/Miscellaneous.css